src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj

```diff
@@ -9,20 +9,13 @@
     <RollForward>LatestPatch</RollForward>
   </PropertyGroup>
 
+  <ItemGroup>
+    <FrameworkListFileClass Include="WindowsFormsIntegration.resources.dll" />
```

Comment:
@LakshanF this change may break publishing self contained app scenario. https://github.com/dotnet/wpf/blob/main/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props doesn't seem to implicitly include this like it does with `WindowsFormsIntegration.dll` and this originally had no profile in .NET 8.

Reply:
Would it be possible to test these scenarios?

Reply:
The `_SDKImplicitReference` code in Microsoft.NET.Sdk.WindowsDesktop.props doesn't apply here, that's for full .NET Framework.  The actual resolution of files in the runtime or targeting packs happen in tasks in the SDK such as `ProcessFrameworkReferences`, `ResolveTargetingPackAssets`, and `ResolveRuntimePackAssets`.